core/core.md

```diff
@@ -8,7 +8,7 @@ In order to understand the Fora protocol, it is useful to define the entities in
 
 **User**: This is a member of the Fora community, they may or may not host the Fora community. A member that hosts the Fora community will specifically be termed **Host** and a member isn't a host and relies on a Host to connect to the community will be specifically termed **Light User**. (NOTE: This is analogous to the Full Node/Light Client distinction in blockchains). Users have a unique name in the community and an associated public key.
 
-**Host**: As mentioned above, a host is a member that maintains the full state of the community. They will receive messages from other users of the community over a gossip network and update their own state, as well as gossip any updates to the rest of the network. They may or may not serve as an access point for Light users. Hosts will listen to events on the ActivityPub network to record new interactions, and they will also listen to events on the blockchain in order to keep track of any decisions made there. An honest Host will delete any banned content from their local copy of the community and ignore any banned user by refusing to accept and propogate their messages.
+**Host**: As mentioned above, a host is a member that maintains the full state of the community. They will receive messages from other users of the community over a gossip network and update their own state, as well as gossip any updates to the rest of the network. They may or may not serve as an access point for Light users. Hosts will listen to events on the blockchain in order to keep track of any decisions made there. An honest Host will delete any banned content from their local copy of the community and ignore any banned user by refusing to accept and propogate their messages.
```

Comment:
I get that we may want to separate out the ActivityPub specifics from the core document. But Hosts must listen to blockchain decisions as well as community interactions. Maybe rephrase to:

```suggestion
**Host**: As mentioned above, a host is a member that maintains the full state of the community. They will receive messages from other users of the community over a gossip network and update their own state, as well as gossip any updates to the rest of the network. They may or may not serve as an access point for Light users. Hosts will record and propogate community interactions, and they must also listen to events on the blockchain in order to abide by any decisions made there. An honest Host will delete any banned content from their local copy of the community and ignore any banned user by refusing to accept and propogate their messages.
```